desc/optimize/optimizer.py

```diff
@@ -614,6 +667,59 @@ def _get_default_tols(
     return stoptol
 
 
+##TODO: Add Diagnostics for x_scale to track what modes are being scaled
+def _create_exponential_spectral_scale(
+    eq, alpha=1.2, scale_type="Linf", min_value=1e-7
+):
+    """Create x_scale using exponential spectral scaling.
+
+    Parameters
+    ----------
+    eq : Equilibrium
```

Comment:
Since right now this is only the surface fourier modes, this should also work if `FourierRZToroidalSurface` is passed in, would just need some extra logic (`R_lmn` for the surface instead of `Rb_lmn` for the eq)